src/pytest_qgis/qgis_interface.py

```diff
@@ -204,13 +213,15 @@ def removeToolBarIcon(self, action: QAction) -> None:
         """
         pass
 
-    def addToolBar(self, name: str) -> None:
+    def addToolBar(self, name: str) -> QToolBar:
         """Add toolbar with specified name.
 
         :param name: Name for the toolbar.
         :type name: str
         """
-        pass
+        toolbar = QToolBar(name, parent=self._mainWindow)
+        self._toolbars.append(toolbar)
```

Comment:
This will fail since self._toolbars is a dictionary. Use:
```suggestion
        self._toolbars[name] = toolbar
```